### PR TITLE
fix broken code link

### LINKED ISSUE #15344: dead link in stitching tutorial
Something weird in[ stitching tutorial](https://docs.opencv.org/master/d8/d19/tutorial_stitcher.html) :  link to code is dead. 

This tutorial code's is shown lines below. You can also download it from [here](https://github.com/opencv/opencv/tree/master/samples/cpp/samples/cpp/stitching.cpp).

### PR DESCRIPTION
**Pull request by prakhar9998**
resolves  #15344

added proper link to the stiching.cpp file

### CODE REVIEW

File: doc/tutorials/stitching/stitcher/stitcher.markdown [hunk 1]
```diff
@@ -15,7 +15,7 @@ Code
 ----
 
 This tutorial code's is shown lines below. You can also download it from
-[here](https://github.com/opencv/opencv/tree/master/samples/cpp/samples/cpp/stitching.cpp).
+[here](https://github.com/opencv/opencv/blob/master/samples/cpp/stitching.cpp).
```
Comment (alalek) [inline doc/tutorials/stitching/stitcher/stitcher.markdown]: Just remove samples/cpp duplication:
https://github.com/opencv/opencv/tree/master/samples/cpp/stitching.cpp

**Comment by LaurentBerger** [comment]
What's about removing the sentence ?

>  If you have cuda device cv::Stitcher can be configured to offload certain operations to GPU. If you prefer this configuration set try_use_gpu to true.

**Comment by LaurentBerger** [comment]
What's about remove the sentence : 

> If you have cuda device cv::Stitcher can be configured to offload certain operations to GPU. If you prefer this configuration set try_use_gpu to true.

File: doc/tutorials/stitching/stitcher/stitcher.markdown [hunk 2]
```diff
@@ -15,7 +15,7 @@ Code
 ----
 
 This tutorial code's is shown lines below. You can also download it from
-[here](https://github.com/opencv/opencv/tree/3.4/samples/cpp/samples/cpp/stitching.cpp).
+[here](https://github.com/opencv/opencv/tree/master/samples/cpp/stitching.cpp).
```
Comment (alalek) [inline doc/tutorials/stitching/stitcher/stitcher.markdown]: Please keep `/3.4/` URLs on 3.4 branch.

Comment (prakhar9998) [reply doc/tutorials/stitching/stitcher/stitcher.markdown]: I've updated it in the new commit

**Pull request merged**